src/lib.rs

```diff
@@ -298,7 +298,7 @@ fn get_num_cpus() -> usize {
     target_os = "fuchsia")
 )]
 fn get_num_cpus() -> usize {
-    let cpus = unsafe { libc::sysconf(libc::_SC_NPROCESSORS_ONLN) };
+    let cpus = unsafe { libc::sysconf(libc::_SC_NPROCESSORS_CONF) };
```

Comment:
This function is used by several targets. Should they all use `CONF` instead of `ONLN`? Or should android be separated?